dist/less/overlay-panel.less

```diff
@@ -132,3 +149,7 @@ body.overlay-open {
     animation: modalSlideUp 0.2s ease-out forwards;
   }
 }
+
+.wizard-overlay .catalogs-overlay-panel {
+  padding-bottom: 0;
```

Comment:
I've assumed that would still be wanted by other uses (non-wizard). I'm OK with this if we feel it is OK to change the generic panel and accommodate the change in other uses if necessary.